test/test_graft.py

```diff
@@ -171,7 +171,7 @@ def test_diamond_translate(self):
                 sample_name = os.path.basename(fasta.name[:-3])
                 orfFile = os.path.join(tmp, sample_name, '%s_orf.fa' % sample_name)
                 expected_aln = sorted('''>HWI-ST1243:121:D1AF9ACXX:8:1101:4036:46970
-NKLFPAKQAGQLKKAIGKTLWQGVHIPTIVSRTCDGGKTSRWVAWQTCM
+NKLFPAKQAGQLKKAIGKTLWQGVHIPTIVSRTCDGGKTSRWVAWQ
```

Comment:
Not sure what has happened here. Current diamond search does not include the "TCM" suffix.

Reply:
OK. Maybe change it then. Might pay to run this test locally and make sure it works first so don't have to keep waiting for actions

Reply:
Oh I confirmed it works. Just not sure what changed to make it fail in the first place.